.github/CODEOWNERS

```diff
@@ -69,6 +69,7 @@ platforms/core-runtime/                                     @gradle/bt-core-runt
 platforms/core-runtime/build-operations/                    @gradle/bt-core-runtime-maintainers @gradle/bt-execution @gradle/bt-dv-build-cache
 platforms/core-runtime/functional/                          @gradle/bt-core-runtime-maintainers @gradle/bt-execution @bamboo
 platforms/core-runtime/files/                               @gradle/bt-core-runtime-maintainers @gradle/bt-execution @gradle/bt-dv-build-cache
+platforms/core-runtime/service-*/                           @gradle/bt-core-runtime-maintainers @alllex
```

Comment:
Strikes me that this is the only case a username is used. Shouldn't we fix your group membership instead?

Reply:
I don't see why personally opting into specific things is something we want to avoid. There is no reason to create a group specifically for services, at least now. And I also wouldn't want to get notified about the full `core-runtime`.

FTR, there are other personal opt-ins (`@bamboo, @ldaley, @tresat`). But if anything, I would encourage more folks in opt in for the areas that care about. The need for this may become more obvious further down the path of Gradle modularization.